resources/Microsoft.NET.SDK.ToolInstaller/Microsoft.NET.SDK.ToolInstaller.psd1

```diff
@@ -0,0 +1,24 @@
+@{
+    RootModule = 'Microsoft.NET.SDK.ToolInstaller.psm1'
+    ModuleVersion = '0.1.0'
+    GUID = '2e883e78-1d91-4d08-9fc1-2a968e31009d'
+    Author = 'Microsoft Corporation'
```

Comment:
@denelon - Is there an ownership concept with DSCs similar to Winget packages?  How do you ensure the validity of the author and that they have sole ownership?

Reply:
Since this is an OSS repository, we'd consider PRs from the community, but since this is a Microsoft "package" the resource would require our approval to make sure we're good with the implementation. Anything published to the PowerShell gallery from here would need that validation before we would sign and publish the module.